jkube-kit/enricher/api/src/main/java/org/eclipse/jkube/kit/enricher/api/visitor/MetadataVisitor.java

```diff
@@ -126,6 +126,13 @@ private static MetaDataConfig getLabels(ResourceConfig resourceConfig) {
     return Optional.ofNullable(resourceConfig).map(ResourceConfig::getLabels).orElse(new MetaDataConfig());
   }
 
+  private String appendTrailingNewLineIfMultiline(String value) {
+    if (value.contains(System.lineSeparator()) && !value.endsWith(System.lineSeparator())) {
```

Comment:
just wondering, what if it ends with `\n `

Reply:
I checked this scenario and found this:

When I provide such configuration in XML configuration, result is unaffected because XML configuration seems to swallow trailing newline https://github.com/eclipse/jkube/issues/1829#issuecomment-1314221042

In the case of Groovy DSL configuration, It seems to generate wrong output like this:
```yaml
  metadata:
    annotations:
      multiline/config: "proxyMetadata:\n ISTIO_META_DNS_CAPTURE: \"false\"\nholdUntilProxyStarts:\
        \ true\n    \n"
```

Not sure if calling `value.trim()` would help since it removes trailing newline as wel.

Reply:
Is there going to be any follow up on this?